src/multi_auth/providers/restream.cr

```diff
@@ -0,0 +1,62 @@
+require "uuid"
+
+class MultiAuth::Provider::Restream < MultiAuth::Provider
+  def authorize_uri(scope = nil)
```

Comment:
According to their docs, they haven't officially implemented scopes in their autorize_uri (yet).
<img width="1289" alt="Screen Shot 2021-12-14 at 1 22 58 PM" src="https://user-images.githubusercontent.com/2391/146081403-0367b80c-350a-4579-b1c3-1a93796b62ae.png">